server/controllers/spreadsheet-controller.js

```diff
@@ -432,6 +433,12 @@ var processGRNmap = function (path, res, app) {
     helpers.attachFileHeaders(res, path);
     network = parseSheet(sheet);
 
+    // Parse expression and 2-column data, then add to network object
+    var additionalData = parseAdditionalSheets(sheet);
+    for (var key in additionalData) {
+        network[key] = additionalData[key];
+    }
```

Comment:
jQuery has an `extend` helper function which can replace this block with a one-liner, I think. Look it up here: https://api.jquery.com/jquery.extend/

Reply:
I don't think we can use jQuery on the server side, but I found `Object.assign()` which does the same thing!